ts/navigation/ProfileNavigator.tsx

```diff
@@ -0,0 +1,104 @@
+import { createStackNavigator } from "@react-navigation/stack";
+import * as React from "react";
+import LogoutScreen from "../components/screens/LogoutScreen";
+import EmailInsertScreen from "../screens/onboarding/EmailInsertScreen";
+import EmailReadScreen from "../screens/onboarding/EmailReadScreen";
+import TosScreen from "../screens/onboarding/TosScreen";
+import CalendarsPreferencesScreen from "../screens/profile/CalendarsPreferencesScreen";
+import CgnLandingPlayground from "../screens/profile/CgnLandingPlayground";
+import DownloadProfileDataScreen from "../screens/profile/DownloadProfileDataScreen";
+import EmailForwardingScreen from "../screens/profile/EmailForwardingScreen";
+import FiscalCodeScreen from "../screens/profile/FiscalCodeScreen";
+import LanguagesPreferencesScreen from "../screens/profile/LanguagesPreferencesScreen";
+import MarkdownPlayground from "../screens/profile/playgrounds/MarkdownPlayground";
+import PrivacyMainScreen from "../screens/profile/PrivacyMainScreen";
+import ProfileDataScreen from "../screens/profile/ProfileDataScreen";
+import RemoveAccountDetails from "../screens/profile/RemoveAccountDetailsScreen";
+import RemoveAccountInfo from "../screens/profile/RemoveAccountInfoScreen";
+import RemoveAccountSuccess from "../screens/profile/RemoveAccountSuccessScreen";
+import SecurityScreen from "../screens/profile/SecurityScreen";
+import ServicesPreferenceScreen from "../screens/profile/ServicesPreferenceScreen";
+import ShareDataScreen from "../screens/profile/ShareDataScreen";
+import WebPlayground from "../screens/profile/WebPlayground";
+import { Showroom } from "../screens/showroom/Showroom";
+import { ProfileParamsList } from "./params/ProfileParamsList";
+import ROUTES from "./routes";
+
+const Stack = createStackNavigator<ProfileParamsList>();
+
+/**
+ * A navigator for all the screens of the Profile section
+ */
+const ProfileStackNavigator = () => (
+  <Stack.Navigator
+    initialRouteName={ROUTES.PROFILE_DATA}
+    headerMode={"none"}
+    screenOptions={{ gestureEnabled: true }}
+  >
+    <Stack.Screen name={ROUTES.PROFILE_DATA} component={ProfileDataScreen} />
+    <Stack.Screen
+      name={ROUTES.PROFILE_PRIVACY_MAIN}
+      component={PrivacyMainScreen}
+    />
+    <Stack.Screen name={ROUTES.PROFILE_PRIVACY} component={TosScreen} />
+    <Stack.Screen
+      name={ROUTES.PROFILE_PRIVACY_SHARE_DATA}
+      component={ShareDataScreen}
+    />
+    <Stack.Screen name={ROUTES.PROFILE_SECURITY} component={SecurityScreen} />
+    <Stack.Screen
```

Comment:
An entry is missing here, which is [this one](https://github.com/pagopa/io-app/pull/4029/files#diff-a4c7a3002810882e29e3429ac0a092256668702299a83e8d4b599dcc1a52cb5cL41-L43):
```tsx
<Stack.Screen
  name={ROUTES.PROFILE_PREFERENCES_HOME}
  component={PreferencesScreen}
/>
```

Reply:
Added in [6f2ab48](https://github.com/pagopa/io-app/pull/4029/commits/6f2ab4869156ac2870513e8bb446c505163111e1) 🙏🏻